app-misc/asciinema/asciinema-2.0.1.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="Terminal session recorder"
+DESCRIPTION="asciinema lets you easily record terminal sessions and replay \
+them in a terminal as well as in a web browser. It is the best companion of \
+asciinema.org."
+HOMEPAGE="https://asciinema.org/
+	https://pypi.org/project/asciinema/
+	https://github.com/asciinema/asciinema"
```

Comment:
Please keep one homepage: https://asciinema.org/

Reply:
@korli, I'd say we can keep the 3 URIs.
The second and third URIs can perfectly be considered as secondary homepages. Some recipes already do that.

Reply:
Okay, one homepage only is enough. Sorry.